.jfrog-pipelines/TFproviderTest.yml

```diff
@@ -139,11 +150,7 @@ pipelines:
             - add_run_variables ARTIFACTORY_ACCESS_TOKEN=${ACCESS_KEY}
             - echo "Unset ARTIFACTORY_PASSWORD, acceptance test will use ARTIFACTORY_ACCESS_TOKEN instead"
             - unset ARTIFACTORY_PASSWORD
-            - export TF_ACC=true
-            - make acceptance
-            # TODO: enable after migration to repo21
-            #- export SONAR_TOKEN=${int_terraform_artifactory_sonarqube_token}
-            #- make scan
+            - TF_ACC=true make acceptance TEST=${directory}
```

Comment:
is `&&` missing here?

Reply:
No need, the `TF_ACC=true` doesn't need `&&` since we aren't executing it as separate commands.